### DISCUSSION TITLE
LBP exchange rate

### DISCUSSION THREAD

**Discussion by marcondy** [btcpayserver/btcpayserver#4001] [General]
Lebanon is struck by hyperinflation for past 3 years, however the official bank rate $1  = 1507 LBP  never changed,while actual (as of writing) is $1 = 29,800 LBP.

I propose to correct the exchange rate using the following source :

lirarate.org[Lirarate](lirarate.org) 

`http://107.170.26.73:8156/~entropy/rates/lirarate`

This will help Lebanese users to implement and show the correct sat / BTC per LBP  values.

**Comment by Kukks** [comment]
The link you posted does not work and the lirarate.org site seems to not have a public api.

**Comment by bolatovumar** [comment]
I tried searching online for public APIs which provide LBP pairs but couldn't find any.

**Reply by bolatovumar** [reply]
@marcondy actually there an exchange rate provider for LBP and I'm adding it here: https://github.com/btcpayserver/btcpayserver/pull/4048
